Week2/homework/greeting.js

```diff
@@ -0,0 +1,8 @@
+const Dutch = 'Hallo Wereld';
+
+const French = 'Bonjour le monde';
+
+const Danish = 'Hej Verden';
+
+console.log(Dutch,fFrench,Danish);
```

Comment:
I think you made a little mistake here:)

Reply:
yes, I have just noticed that :) and corrected it